solr/contrib/ltr/src/java/org/apache/solr/ltr/LTRRescorer.java

```diff
@@ -186,18 +191,40 @@ public void scoreFeatures(IndexSearcher indexSearcher,
         docBase = readerContext.docBase;
         scorer = modelWeight.scorer(readerContext);
       }
-      scoreSingleHit(indexSearcher, topN, modelWeight, docBase, hitUpto, hit, docID, scoringQuery, scorer, reranked);
+      scoreSingleHit(
```

Comment:
ugh, these are both bad, not sure what I think we should do here.

Reply:
hmm, yeah, ten arguments for the method is also quite a lot, perhaps a different method signature somehow with an intermediate structure or something. would need to give it some thought (beyond reformatting code that is)

Reply:
#192 would be one way to split the 10-args method.